website/static/js/share/index.js

```diff
@@ -87,6 +92,14 @@ ShareApp.controller = function() {
 
     });
 
+    m.request({
+        method: 'GET',
+        url: '/api/v1/share/documents/' // TODO where will the postgres API live??
+    }).then(function(data) {
+        self.vm.rawNormedLoaded(true);
+    }, function(err) {
+        // blank, but needed to handle missing rawNormed API
```

Comment:
The error cannot be fully suppressed, as far as I can tell. It's a core part of Javascript. That said, it doesn't halt execution. The other option would be to remove the call until the endpoint is implemented.

I don't think it's a problem leaving it the way it is, but I'm not opposed to temporarily commenting or removing it. No strong opinion.